lib/src/Uncertainty/Algorithm/WeightedExperiments/LHSResult.cxx

```diff
@@ -218,6 +235,8 @@ Graph LHSResult::drawHistoryCriterion(UnsignedInteger restart, const String & ti
 
 Graph LHSResult::drawHistoryTemperature(const String & title) const
 {
+  if (!collAlgoHistory_.getSize())
+    throw InvalidArgumentException(HERE) << "No design stored!";
```

Comment:
same here

Reply:
The argument of the method is the graph title. We are using `CollAlgoHistory_[optimalIndex_]` via `drawHistoryTemperature(UnsignedInteger restart, const String & title)`.